test/TesterSQLUtils/SqlStatisticsPublisherTests/StatisticsPublisherProviderRuntime.cs

```diff
@@ -8,16 +9,9 @@ namespace UnitTests.SqlStatisticsPublisherTests
 {
     internal class StatisticsPublisherProviderRuntime : IProviderRuntime
     {
-        private readonly Logger logger;
-
-        public StatisticsPublisherProviderRuntime(Logger logger)
-        {
-            this.logger = logger;
-        }
-
         public Logger GetLogger(string loggerName)
```

Comment:
this method can be removed now, is that correct?

Reply:
That method is from IProviderRuntime I think, which we agree to keep to the interface, or at least keep in a legacy package by using extension method. We can deal with it in next PR.